packages/peregrine/lib/targets/TalonWrapperConfig.js

```diff
@@ -33,6 +33,52 @@ class TalonWrapperConfig {
              */
             useApp: new WrappableTalon(addTransforms, 'App/useApp')
         };
+        /**
+         * Provides access to the talon used in Venia's Checkout UI component.
+         * @instance
+         */
+        this.Checkout = {
```

Comment:
In the latest versions of Venia, the drawer checkout and its talons here are essentially deprecated, in favor of `CheckoutPage`. Could we target `CheckoutPage` talons instead?

Reply:
In the newest commit, I changed this to CheckoutPage. Thanks!